### ISSUE TITLE
RRmpWald $freqvec output is formatted differently depending on input type

### ISSUE THREAD

**Issue by mariev** [ABS-dev/PF#16]
Current behavior (from help page) is:

```r
ex1 <- RRmpWald(pos ~ tx + cluster(cage), New, compare = c('con', 'vac'))
ex1$freqvec
## pos pos neg pos pos neg neg neg 
##       7      13       2       4 

require(magrittr)
thistable <- New %>%
  tidyr::spread(tx, pos) %>%
  dplyr::mutate(vac = factor(vac, levels = 1:0),
    con = factor(con, levels = 1:0)) %>%
  with(., table(vac, con))
ex2 <- RRmpWald(x = as.vector(thistable))
ex2$freqvec
## [1]  7 13  2  4
```

verify which one is preferred

**Comment by mariev** [comment]
$freqvec appears to be one of three representations of the data that is preserved in the output

```r
ex1$freqvec
## pos pos neg pos pos neg neg neg 
##       7      13       2       4 
ex1$xtable
##      [,1] [,2]
## [1,]    7    2
## [2,]   13    4

ex1$multvec
##   vac con Freq
## 1 pos pos    7
## 2 neg pos   13
## 3 pos neg    2
## 4 neg neg    4
```

**Comment by mariev** [comment]
initial comments from @hydrodynamicstability:

re: `ex2`

> it looks to me like the order is “vac con”.  This is the reverse order in which the user specifies the group names, which is unexpected.  To address informativeness properly, one may need to use paste() to create strings like “vac_pos con_pos”, “vac_neg con_pos”, etc., where it pulls the names from the compare vector.

re: #21 

>The $multvec slot would seem to make the $freqvec and $xtable slots redundant and unnecessary.  I wonder if those are vestiges of the debugging process.  If so, it might be helpful to maintain them for developer use, but steer users to $multvec.  RRmpWald is a relatively new and relatively untested function compared to most of the others in the package.

**Comment by mariev** [comment]
resolved with  #21
